packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -94,6 +94,10 @@ export class HdPaths {
     return HdPaths.bip44(HdPaths.coinTypes.eth, 0, 0, account);
   }
 
+  public static cosmos(account: number): readonly Slip10RawIndex[] {
+    return HdPaths.bip44(HdPaths.coinTypes.atom, 0, 0, account);
```

Comment:
Do we have a documentation link for this path? My usual source (https://github.com/trezor/trezor-firmware/blob/master/core/docs/misc/coins-bip44-paths.md) does not list Cosmos

Reply:
- Coin type is listed here: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
- Account we're choosing to set to 0 for simplicity.
- Change address doesn't apply (so 0).
- `account` is really address index.

See https://github.com/cosmos/cosmos-sdk/blob/2c96bbb/crypto/keys/types.go#L34-L36

Reply:
That cosmos SDK link is very helpful. There however, a two dimentional space is create (`account` and `index` but let's ignore the naming of the axes for now). In https://github.com/cosmos/cosmos-sdk/blob/v0.37.4/client/keys/add.go#L39-L259 it looks like both dimensions are used.

It would be interesting to find out what wallets typically do.

Reply:
- Cosmostation uses the same: https://github.com/cosmostation/cosmosjs/blob/master/src/index.js#L20
- As does Lunie: https://github.com/luniehq/cosmos-keys/blob/7fe5bb4b85d40b6272cfccc12f1ff3a5f0ee380e/src/cosmos-keys.ts#L8

Reply:
Finally a confirmation here! https://github.com/cosmos/cosmos-sdk/issues/4278#issuecomment-561238038